index.html

```diff
@@ -380,12 +385,14 @@ <h2>Glossary</h2>
 
 <p><dfn data-lt="unicode locale|unicode locale identifier|unicode locale identifiers|unicode locales|Unicode Locale Identifier" id="def_unicode_locale" class="lint-ignore">Unicode Locale Identifier</dfn> or <em>Unicode Locale</em>. A <a href="#def_language_tag" class="termref">language tag</a> that follows the additional rules and restrictions on subtag choice defined in UTR#35 [[UAX35]]. Any valid Unicode locale identifier is also a <a href="#def_valid" class="termref">valid</a> [[BCP47]] <a href="#def_language_tag" class="termref">language tag</a>, but a few <a href="#def_valid" class="termref">valid language tags</a> are not also valid Unicode locale identifiers.</p>
 
+<p><dfn data-lt="universal character set|unicode" id="def_universal_character_set" class="lint-ignore">Universal Character Set</dfn> or <em>Unicode</em>. The character set defined by the [[Unicode]] Standard and which includes all of the characters used to encode text, including historical or extinct writing systems as well as modern usage, private use, typesetting symbols, and other things, such as the emoji. Other <a>character sets</a> are defined subsets of Unicode.</p>
```

Comment:
defined subsets seems to imply that they are designed to be subsets

maybe drop 'defined'

Reply:
This is copied from specdev. Happy to remove "defined"